### PR TITLE
migration: Update error message

### PR DESCRIPTION
**Pull request by cliping**

### CODE REVIEW

**Comment by cliping** [comment]
Before:

TestFail: Can not find the expected patterns 'Path .* is not accessible: No such file or directory' in output 'error: Cannot access storage file '/var/lib/avocado/data/avocado-vt/images/readonly.raw': No such file or directory'

After:
 (1/1) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.autoskipped_disks.disk_with_readonly.copy_storage_all.local_storage.p2p: PASS (129.94 s)

File: libvirt/tests/cfg/migration_with_copy_storage/autoskipped_disks.cfg [hunk 1]
```diff
@@ -59,7 +59,7 @@
             second_disk_name = "readonly.raw"
             local_storage:
                  status_error = "yes"
-                 err_msg = "Path .* is not accessible: No such file or directory"
+                 err_msg = "Path .* is not accessible: No such file or directory|Cannot access storage file.*: No such file or directory"
```
Comment (dzhengfy) [inline libvirt/tests/cfg/migration_with_copy_storage/autoskipped_disks.cfg:62]: I do think no need to match many detailed words. 
Maybe we just use `No such file or directory` which I think it is root cause

Comment (cliping) [reply libvirt/tests/cfg/migration_with_copy_storage/autoskipped_disks.cfg]: Updated, thanks.

**Review by Yingshun — APPROVED** [review]

**Comment by Yingshun** [comment]
@dzhengfy It's a minor fix and addressed your comments, hence merging it.

**Pull request merged**